bump filestack version

nolman opened the pull request:
I had to use some of the newer filestack features and needed an upgraded filestack version. I don't believe there are any breaking changes in the API.

nolman commented:
Added in a `filestack-image` helper /w tests as well so that one can easily generate filestack image links

mminkoff commented on tests/integration/helpers/filestack-image-test.js:
```diff
@@ -0,0 +1,41 @@
+
+test('it handles missing hash arguments', function(assert) {
+  this.set('imageToken', '123ABC');
+  this.render(hbs`{{filestack-image imageToken width='200' height='300' fit=crop}}`);
```
Unless I misunderstand something, `crop===undefined` here, which is what you're trying to test but isn't obvious.  Perhaps set `const crop = null` first, or just say `fit=null`?

mminkoff commented on addon/helpers/filestack-image.js:
```diff
@@ -0,0 +1,48 @@
+    ].filter((element) => {
+      return element;
+    }).join('/')
+    // return .compact().join('/')
```
this comment should be removed, right?

nolman commented:
One of my co-founders (@scudco) beat me to addressing your feedback @mminkoff. Thanks again for making this library, let us know if you would like anything else taken care of with this PR.

mminkoff commented:
Version 0.0.8 is available for your installing pleasure, @nolman @scudco

mminkoff commented:
@nolman @scudco come to think of it, adding a little instruction about the new helper to the README would be nice - want to add something, or even just send me some text and I'll add it?
